querybook/server/lib/result_store/stores/file_store.py

```diff
@@ -61,9 +62,14 @@ def start(self):
         pass
 
     def get_csv_iter(self, number_of_lines: Optional[int]):
-        with open(self.uri) as result_file:
-            reader = csv.reader(result_file)
-            return islice(reader, number_of_lines)
+        # with open(self.uri) as result_file:
+        #     reader = csv.reader(result_file)
+        #     return islice(reader, number_of_lines)
+
+        # Execution completes but was showing error "I/O exception opening closed file" while displaying result. 
+        # Commenting the above and adding the below as fix.
```

Comment:
please remove all those commented-out lines, they not needed with the new implementation.

Reply:
done

Reply:
Hi @jczhong84,
I have fixed the lint errors too.